### PR TITLE
enhance the cli to generate doc json file

### PR DESCRIPTION
**Pull request by njzjz**
waits deepmodeling/dargs#4 reviewed and merged, and new version of dargs published.

### CODE REVIEW

**Comment by njzjz** [comment]
@y1xiaoc A new version of dargs is required to be published.

**Comment by y1xiaoc** [comment]
new darg version has been published

File: deepmd/entrypoints/doc.py [hunk 1]
```diff
@@ -1,11 +1,16 @@
 """Module that prints train input arguments docstrings."""
 
-from deepmd.utils.argcheck import gen_doc
+from deepmd.utils.argcheck import gen_doc, gen_json
 
 __all__ = ["doc_train_input"]
 
 
-def doc_train_input():
+def doc_train_input(*, out_type: bool = "rst", **kwargs):
```
Comment (amcadmus) [inline deepmd/entrypoints/doc.py:8]: shouldn't `out_type` be `str`?

Comment (njzjz) [reply deepmd/entrypoints/doc.py:8]: yes

**Review by amcadmus — APPROVED** [review]

**Pull request merged**